routes/movies.routes.js

```diff
@@ -0,0 +1,97 @@
+// starter code in both routes/celebrities.routes.js and routes/movies.routes.js
+const router = require("express").Router();
+const Celebrity = require("../models/Celebrity.model");
+const Movie = require("../models/Movies.model")
+
+// all your routes here
+
+router.get("/movies/create", async (req, res, next)=> { 
+    try {
+        const celebrities = Celebrity.find()
```

Comment:
You missed the await in here!
`const celebrities = await Celebrity.find()`

Remember that if you're using `async` - you need - `await` !